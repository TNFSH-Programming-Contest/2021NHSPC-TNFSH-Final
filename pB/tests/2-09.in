8 15 4
9 12 2 13 7 5
4 2 13 3 1 12
7 5 1 6 13 4
9 1 11 14 6 5
1 6 8 12 4 13
12 6 7 10 13 2
7 5 12 11 8 9
14 1 10 9 3 11
5 2 4 5 3 8
3 6 2 1
2 8 2
3 3 2 5
4 3 1 7 4
4 3 6 5 4
4 1 7 3 6
2 5 7
4 8 7 1 4
2 6 8
3 8 7 4
5 7 1 5 6 2
5 1 3 2 6 5
2 4 8
0
