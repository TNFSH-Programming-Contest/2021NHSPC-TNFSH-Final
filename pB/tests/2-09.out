1 5
0
0
1 2
0
0
2 3 7
0
2 1 4
0
0
1 6
0
1 8
0
